### PR TITLE
Fix recording of Physical Input Read Time in operator stats

### PR DESCRIPTION
**Pull request by raunaqmorarka**
## Description

Fix recording of Physical Input Read Time in operator stats

> Is this change a fix, improvement, new feature, refactoring, or other?

fix
> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

core query engine
> How would you describe this change to a non-technical end user or system administrator?

fixes display of Physical Input Read Time in UI
## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: core/trino-main/src/main/java/io/trino/operator/DriverContext.java [hunk 1]
```diff
@@ -346,7 +346,7 @@ public DriverStats getDriverStats()
         if (inputOperator != null) {
             physicalInputDataSize = inputOperator.getPhysicalInputDataSize();
             physicalInputPositions = inputOperator.getPhysicalInputPositions();
-            physicalInputReadTime = inputOperator.getAddInputWall();
+            physicalInputReadTime = inputOperator.getPhysicalInputReadTime();
 
             internalNetworkInputDataSize = inputOperator.getInternalNetworkInputDataSize();
             internalNetworkInputPositions = inputOperator.getInternalNetworkInputPositions();
```
Comment (sopel39) [inline core/trino-main/src/main/java/io/trino/operator/DriverContext.java:352]: I don't think `internalNetworkInputReadTime = inputOperator.getAddInputWall();` is ever used? Could we just remove it and add a TODO (+ github issue)? Alternatively you could just fix it

**Review by sopel39 — APPROVED** [review]
lgtm % comment

File: core/trino-main/src/main/java/io/trino/operator/DriverContext.java (hunk 1 above)
Comment (raunaqmorarka) [reply core/trino-main/src/main/java/io/trino/operator/DriverContext.java:351]: Added a commit to remove internalNetworkInputReadTime

File: core/trino-main/src/main/java/io/trino/operator/OperatorStats.java [hunk 2]
```diff
@@ -49,6 +49,8 @@
     private final Duration addInputCpu;
     private final DataSize physicalInputDataSize;
     private final long physicalInputPositions;
+
```
Comment (sopel39) [inline core/trino-main/src/main/java/io/trino/operator/OperatorStats.java:52]: remove empty line

**Review by sopel39 — APPROVED** [review]
lgtm % comment

File: core/trino-main/src/main/java/io/trino/operator/OperatorStats.java (hunk 2 above)
Comment (raunaqmorarka) [reply core/trino-main/src/main/java/io/trino/operator/OperatorStats.java]: done

**Review by sopel39 — APPROVED** [review]

**Pull request merged**